Test/algo_failure_test.py

```diff
@@ -1,34 +1,35 @@
 import sys
-from multiprocessing import Process
-# look in ../ BEFORE trying to import Algorithmia.  If you append to the
-# you will load the version installed on the computer.
-sys.path = ['../'] + sys.path
 
-import unittest
-import Algorithmia
-import uvicorn
-import time
-from requests import Response
-from Test.api import app
+if sys.version_info[0] >= 3:
```

Comment:
To keep things as simple as possible, if the version of python is less than 3 (aka 2.7); we essentially skip this entire test case, as it does require uvicorn which is not 2.7 compatible.